beacon-chain/beacon-network.md

```diff
@@ -151,35 +151,45 @@ content_key                = selector + SSZ.serialize(light_client_bootstrap_key
 light_client_update_keys   = Container(start_period: uint64, count: uint64)
 selector                   = 0x01
 
-content                    = SSZList(ForkDigest || LightClientUpdate, max_lenght=MAX_REQUEST_LIGHT_CLIENT_UPDATES)
+content                    = SSZList(ForkDigest + LightClientUpdate, max_lenght=MAX_REQUEST_LIGHT_CLIENT_UPDATES)
 content_key                = selector + SSZ.serialize(light_client_update_keys)
 ```
 
 #### LightClientFinalityUpdate
 
 ```
-light_client_finality_update_key  = 0 (uint8)
+light_client_finality_update_key  = Container(optimistic_slot: uint64, finalized_slot: uint64)
```

Comment:
I'm not sure if I understand why do we need an `optimistic_slot` for the finality update?  If I'm not missing something, a `LightClientFinalityUpdate` includes `finalized_header` which includes the finalized slot?

Reply:
There is also the `attested_header` for which we could indicate the slot to indicate the key.
But yes, I am not sure if this is really needed right now, and from my understanding, 1 of the two slots is probably fine, see also my comment: https://github.com/ethereum/portal-network-specs/pull/217#discussion_r1258331128

The finalized slot probably makes more sense then.